src/components/button/button.test.tsx

```diff
@@ -159,6 +159,30 @@ test("sets the correct height when 'services' is passed to the 'buttonType' prop
   expect(icon).toHaveStyle("height: 6px");
 });
 
+/* Styling test for coverage */
```

Comment:
**comment** (non-blocking): I assume these are needed now because they were previously being covered by tests for `ButtonMinor` that you've removed.

I'm happy for these tests to be added to `Button` (unless they're already covered by playwright tests - which they might be), but I would prefer to also:
1) add similar tests explicitly to `ButtonMinor` so that we know for sure that the `size` and `iconType` props from ButtonMinor is getting passed down to Button
2) I'd personally remove the "for coverage" comments as I don't think those are adding very much

Reply:
Same as above, as these are mainly for coverage and styles are captured chromatic so I didn't think it was worth doubling up.